test/ex_doc/autolink_test.exs

```diff
@@ -161,11 +161,30 @@ defmodule ExDoc.AutolinkTest do
       assert autolink(~m"[custom text](`:lists.all/2`)") ==
                ~m"[custom text](http://www.erlang.org/doc/man/lists.html#all-2)"
 
-      # TODO: with custom links and backticks there are no false positives (you
-      #       always mean to link) so we should always warn on mismatches?
-      #       Though backticks are markdown specific, is that ok?
-      # assert_warn(fn ->
-      assert_unchanged(~m"[custom text](`Unknown`)")
+      captured =
+        assert_warn(fn ->
+          assert_unchanged(~m"[custom text](`Unknown`)")
+        end)
+
+      assert captured =~
+               "documentation references module \"Unknown\" but it doesn't exist"
+
+      captured =
+        assert_warn(fn ->
+          assert_unchanged(~m"[custom text](Unknown)")
+        end)
+
+      # File LICENSE exists, but backticks are meant for modules
```

Comment:
I think this comment is misplaced, I'd either put it on line 183 or simply remove it.

Reply:
removed.